### ISSUE TITLE
Transcriber Registration page — improve accessibility & semantics

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#1320]
This is a public page and so should be accessible to all, irrespective of whether or not they are capable of being a transcriber.

The main issue is that checkboxes should be tied by the markup to their respective labels.

**Comment by AlOneill** [comment]
This is how the documents section of the page looks with the markup improved for accessibility. Is this OK or does anything need tweaking visually?

![Transcriber_Registration___FreeReg.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5609b1960367a9170003b106/125516c66e82c728ace21e0d46dc869b2d9180bb86ebba519b6ff593205533612810aa247ebadc1799507e043d0a19be5d4a1f06a3a3dc78e2a436628f5f5ebe8038567dc23511abea871fbd01ba21881a85b5ea9850d2aa6c7bb3d4d216744b470a83ed5f35aa984ec85ff74fe84453a7103761f43b.png)

**Comment by Sherlock21** [comment]
As I  cant get to see the complete document in its context, I am just thinking aloud - and if the topic is covered elsewhaere on these pages, then sorry.

But:
this is the Volunteer Sign ON Agreement.

is there a matching “Resign” or Leave document.? or does one just ‘drop out’

Eric B

**Comment by edickens** [comment]
If someone "Resigns" or just ceases transcribing, we cannot delete their UserID as it is tagged to Batches.  The SC should Disable the UserID
But if they have never uploaded, then they should email their SC to say that they are quitting.   The SC should email me as only I, at present, can delete as it is final and could delete batches.

**Comment by AlOneill** [comment]
@PatReynolds How important is it to have an advert in the main page contents?

I ask because with the ad, there seems to be no good way to ensure that the page flows/tabs well on all devices/screen widths.

**Comment by AlOneill** [comment]
@PatReynolds Unless we place it at the bottom of the page, just before the footer

**Comment by AlOneill** [comment]
@PatReynolds Turns out, it *can* be done, with some extra markup.

**Comment by PatReynolds** [comment]
Longer term important, but short term not- there are other ways to increase revenue, and other pressing needs for accessibility, etc. etc.

**Comment by AlOneill** [comment]
Thanks, Pat. Will keep advert using adjusted markup mentioned above.

Now that I've looked at the ERB file properly (and submitted an empty form), I think the error message styling could do with some attention. In progress ...

**Comment by AlOneill** [comment]
Made a mistake — can't see a way to keep ad in same place visually whilst removing it from flow of form (so that it does not annoy/confuse a user who is tabbing through the form). Ad now placed after the form, where it could be made wider, if desired.

Edited ERB file now with Vino ...

**Comment by AlOneill** [comment]
Revised page can now be viewed on test3: 
https://test3.freereg.org.uk/userid_details/transcriber_registration

It passes WAVE validation (no Errors). 

However, there are **Alerts** re our use of Word/PDF documents which may or may not be accessible: such documents are beyond my current skill set to evaluate/fix.

**Comment by AlOneill** [comment]
Vino is investigating the disappearance of the Placeholder text when a form is submitted with errors ...

**Comment by Vino-S** [comment]
Fixed the Placeholder issue and updated in test3. I had also replaced the html <input> tag with rails :text_field_tag(keeping all the other html options like class, id same)
